client/src/pages/locations/GeoLocation.js

```diff
@@ -213,7 +222,12 @@ const LatLonFormField = ({
       </>
     )
   }
-
+  const setParsedLatLng = (_lat, _lng) => {
```

Comment:
Why the underscores in the variable names?

Reply:
Props are also named lat, lng. I wanted it to be clearly different, not confusing.

Reply:
Seeing _ in a variable name makes me think, ow, does this have something to do with private variables or is there something special about it? I would just call it latVal, lngVal.